### PR TITLE
update the status page for outage

### PR DESCRIPTION
**Pull request by jogendrakmjangid**

### CODE REVIEW

File: psgallery_status.md [hunk 1]
```diff
@@ -3,7 +3,13 @@ PowerShell Gallery Status
 =========================
 ### The [PowerShell Gallery](https://powershellgallery.com) is currently:  __Operating with Statistic Errors__
 
-### 10/15/2020 - PowerShellGallery.com may temporarily experience latency and/or low availability 
+30/10/2020 - PowerShellGallery.com has CDN is down preventing packages from being downloaded and availability for the gallery itself has been extremely low
```
Comment (si458) [inline psgallery_status.md:6]: `PowerShellGallery.com has CDN id down` should read `PowerShellGallery.com CDN is down`

**Review by jogendrajangid1 — COMMENTED** [review]
Grammar correction  in the subject text.

File: psgallery_status.md [hunk 2]
```diff
@@ -3,7 +3,13 @@ PowerShell Gallery Status
 =========================
 ### The [PowerShell Gallery](https://powershellgallery.com) is currently:  __Operating with Statistic Errors__
```
Comment (MattJeanes) [inline psgallery_status.md:4]: This line should say Total Outage or similar

**Review by mudanzasIPROM — APPROVED** [review]
bbbbb

**Pull request closed without merging**